keentools/geotracker/camera_input.py

```diff
@@ -70,16 +70,28 @@ def image_size(self) -> Tuple[int, int]:
 
 class GTGeoInput(pkt_module().GeoInputI):
     _previous_val: int = 0
+    _hash_counter: int = 0
 
     @classmethod
-    def _set_previous_val(cls, val):
+    def increment_hash(cls) -> None:
+        cls._hash_counter += 1
+        if cls._hash_counter > 1000:
+            cls._hash_counter = 0
+
+    @classmethod
+    def get_hash_counter(cls) -> int:
+        return cls._hash_counter
+
+    @classmethod
+    def _set_previous_val(cls, val) -> None:
         cls._previous_val = val
 
     def _rounded(self, val: float) -> int:
         p = frexp(val)
         return int(round(p[0] * 10000, 0) + 3 * p[1])
 
     def geo_hash(self) -> Any:
+        _log.output(_log.color('magenta', 'get geo_hash'))
```

Comment:
Remove this?

Reply:
Here and in other places? Or this is hidden?

Reply:
This call is visible in debug mode only